packages/alfa-rules/src/sia-r61/rule.ts

```diff
@@ -0,0 +1,62 @@
+import { Rule } from "@siteimprove/alfa-act";
+import * as Accessible from "@siteimprove/alfa-aria";
+import { Document, Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { expectation } from "../common/expectation";
+import { hasChild } from "../common/predicate/has-child";
+import { hasName } from "../common/predicate/has-name";
+import { hasRole } from "../common/predicate/has-role";
+import { isDocumentElement } from "../common/predicate/is-document-element";
+import equals = Predicate.equals;
+
+const { and } = Predicate;
+
+export default Rule.Atomic.of<Page, Document>({
+  uri: "https://siteimprove.github.io/sanshikan/rules/sia-r61.html",
+  evaluate({ device, document }) {
+    let firstHeading: Element;
+
+    return {
+      applicability() {
+        if (hasChild(and(Element.isElement, isDocumentElement()))(document)) {
+          const myFirstHeading = document
+            .descendants({ flattened: true })
+            .filter(and(Element.isElement, hasRole(hasName(equals("heading")))))
+            .first();
+          if (myFirstHeading.isSome()) {
+            firstHeading = myFirstHeading.get();
+            return [document];
+          }
+        }
+        return [];
+      },
+
+      expectations(target) {
+        return {
+          1: expectation(
+            Accessible.Node.from(firstHeading, device).every((accNode) =>
```

Comment:
As far as I understand, `every` here means "for every browser in te browser specific list that we get".

Do we have a way to instead map the test and return browser specific outcome?